patroni/ctl.py

```diff
@@ -445,6 +445,31 @@ def parse_scheduled(scheduled):
     return None
 
 
+@ctl.command('reload', help='Reload cluster member configuration')
+@click.argument('cluster_name')
+@click.argument('member_names', nargs=-1)
+@click.option('--role', '-r', help='Reload only members with this role', default='any',
+              type=click.Choice(['master', 'replica', 'any']))
+@option_force
+@click.pass_obj
+def reload(obj, cluster_name, member_names, force, role):
+    cluster = get_dcs(obj, cluster_name).get_cluster()
+
+    members = get_members(cluster, cluster_name, member_names, role, force, 'reload')
+
+    content = {}
+    for member in members:
+        r = request_patroni(member, 'post', 'reload', content, auth_header(obj))
+        if r.status_code == 200:
+            click.echo('Success: reload on member {0}'.format(member.name))
```

Comment:
It returns 200 if there are no changes to apply from yaml file: https://github.com/zalando/patroni/blob/master/patroni/api.py#L184

Reply:
Should we say no changes to apply then?

Reply:
And would it make sense to say "Success" if there are no changes to apply? It isn't a failure. Is there some kind of "noop" prefix you'd prefer I use?

Reply:
Well, it is definitely not an error, so I think something like "Nothing changed in Patroni config on {0}" should be good enough.